packages/grafana-data/src/dataframe/processDataFrame.ts

```diff
@@ -288,6 +320,16 @@ export const toLegacyResponseData = (frame: DataFrame): TimeSeries | TableData =
     }
   }
 
+  if (frame.meta && frame.meta.json) {
+    return {
+      alias: fields[0].name || frame.name,
+      target: fields[0].name || frame.name,
+      datapoints: fields[0].values.toArray(),
+      filterable: fields[0].config ? fields[0].config.filterable : undefined,
+      type: 'docs',
```

Comment:
This isn't really a `TimeSeries`, but not sure if this is okay or not?

Reply:
Think this is ok for now until we make table panel work with DataFrame directly